src/components/tile-select/tile-select.spec.js

```diff
@@ -144,6 +148,62 @@ describe("TileSelect", () => {
       );
     });
   });
+
+  describe("customActionButton render prop", () => {
+    it("clicking it invokes passed onChange callback", () => {
+      const onChangeMock = jest.fn();
+      render({
+        onChange: onChangeMock,
+        checked: true,
+        id: "id",
+        name: "name",
+        customActionButton: (onClick) => <Button onClick={onClick}>Foo</Button>,
+      });
+      wrapper.find(StyledDeselectWrapper).find(Button).prop("onClick")();
```

Comment:
You could also check if `onClick` has been called

Reply:
The `onClick` calls the `onChange` callback in the `TileSelect` so I decided not to bother adding the extra assertion